galeracluster/source/monitoringthecluster.rst

```diff
@@ -1,305 +1,288 @@
+
+- :ref:`wsrep_cluster_size <wsrep_cluster_size>` shows the number of nodes in the cluster, which you can use to determine if any are missing.
+
+  .. code-block:: mysql
+
+     SHOW GLOBAL STATUS LIKE 'wsrep_cluster_size';
+
+     +--------------------+-------+
+     | Variable_name      | Value |
+     +--------------------+-------+
+     | wsrep_cluster_size | 15    |
+     +--------------------+-------+
+
+  You can run this check on any node.  When the check returns a value lower than the number of nodes in your cluster, it means that some nodes have lost network connectivity or they have failed.
+
+- :ref:`wsrep_cluster_status <wsrep_cluster_status>` shows the primary status of the cluster component that the node is in, which you can use in determining whether your cluster is experiencing a partition.
+
+  .. code-block:: mysql
+
+     SHOW GLOBAL STATUS LIKE 'wsrep_cluster_status';
+
+     +----------------------+---------+
+     | Variable_name        | Value   |
+     +----------------------+---------+
+     | wsrep_cluster_status | Primary |
+     +----------------------+---------+
+
+  The node should only return a value of ``Primary``.  Any other value indicates that the node is part of a nonoperational component.  This occurs in cases of multiple membership changes that results in a loss of quorum or in cases of split-brain situations.
+
+  .. seealso:: In the event that you check all nodes in your cluster and find none that return a value of ``Primary``, see :doc:`quorumreset`.
+
+When these status variable check out and return the desired results on each node, the cluster is up and has integrity.  What this means is that replication is able to occur normally on every node.  The next step then is :ref:`checking node status <check-node-status>` to ensure that they are all in working order and able to receive write-sets.
+	       
+  
+      
+
+---------------------------------
+ Checking the Node Status
+---------------------------------
+.. _`check-node-status`:
+
+.. index::
+   pair: Parameters; wsrep_cluster_address
+
+.. index::
+   pair: Parameters; wsrep_ready
+
+.. index::
+   pair: Parameters; wsrep_connected
+
+.. index::
+   pair: Parameters; wsrep_local_state_comment
+
+In addition to checking cluster integrity, you can also monitor the status of individual nodes.  This shows whether nodes receive and process the SQL load from the cluster write-sets and can indicate problems that may prevent replication.
+
+- :ref:`wsrep_ready <wsrep_ready>` shows whether the node can accept queries.
+
+  .. code-block:: mysql
+
+     SHOW GLOBAL STATUS LIKE 'wsrep_ready';
+
+     +---------------+-------+
+     | Variable_name | Value |
+     +---------------+-------+
+     | wsrep_ready   | TRUE  |
+     +---------------+-------+
+
+  When the node returns a value of ``TRUE`` it can accept the SQL load of write-sets from the cluster.  When it returns the value ``FALSE``, almost all queries fail with the error:
```

Comment:
Do not confuse SQL queries coming from clients with write-sets coming from other cluster members. In this case, it is about SQL queries from clients.